### PR TITLE
PRS13446 13445

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record, update

### CODE REVIEW

File: new/PRS13446GabraKrestos.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ገብረ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Gabra Krǝstos</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="academic"/>
```
Comment (thea-m) [inline new/PRS13446GabraKrestos.xml:51]: "academic"? The definition is "Occupation within an academic institution or independently pursuing academic interests". Is there some evidence for this in the manuscript?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/PRS13446GabraKrestos.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS13446GabraKrestos.xml:51]: no, sorry. corrected.

**Review by thea-m — APPROVED** [review]
Sorry, just to make sure: the deletion of the relations is intended?

**Comment by DenisNosnitsin1970** [comment]
Yes, they unlikely were a couple. :)

**Pull request merged**